rules_building_block/execution_github_repo_interaction_from_newly_added_member.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2023/10/11"
+integration = ["github"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/10/11"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+description = """
+A new GitHub Organization member added in the last 7 days has interacted with a repository.
+"""
+from = "now-9m"
+index = ["logs-github.audit-*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Repo Interaction From a Newly Added Member"
+risk_score = 21
+rule_id = "d4fcb2c3-c55e-4485-ad29-e75727b241c5"
+severity = "low"
+tags = ["Domain: Cloud",
+        "Use Case: Threat Detection",
+        "Tactic: Execution",
+        "Rule Type: BBR",
+        "Data Source: Github"
+        ]
+timestamp_override = "event.ingested"
+type = "eql"
+building_block_type = "default"
+
+query = '''
+sequence by github.org with maxspan=7d
```

Comment:
A seven-day sequence rule seems very performance-intense. Also, I am not sure whether this one adds that much additional coverage (taking into account the new terms repo interaction rules already) that we can disregard the performance. Maybe someone who knows more about performance than me could comment on this one.

Reply:
I agree, the new terms repo interaction rule will technically catch this so it may not be worth the performance to create this one.